### ISSUE TITLE
Remove "sklearn" from requirements

### ISSUE THREAD

**Issue by melanopsis** [bmabey/pyLDAvis#234]
`sklearn` is being deprecated. Please see https://github.com/scikit-learn/sklearn-pypi-package.

**Comment by georgakd** [comment]
Is there any update on this issue? It really causes us issues in our internal NLP LDA project.

**Comment by ahmaddynugroho** [comment]
here's how i manually fix this:

1. download the package from https://pypi.org/project/pyLDAvis/#files
2. extract it and locate `requirements.txt`
3. delete `sklearn` line
4. now you can use it. because i'm using Poetry i run `poetry add modified_pyldavis_folder`
5. done

i was trying to make a pull request but it requires a test

**Comment by bmabey** [comment]
@msusol is this something you plan on tackling? Otherwise I'd accept a PR from anyone on this thread.

**Comment by matthewkdies** [comment]
Adding a bump here because this issue is affecting me as well. I can take a stab at it if we don't hear further from Mark.

**Comment by matthewkdies** [comment]
I went ahead and created a PR for this regardless, here's the link here: https://github.com/bmabey/pyLDAvis/pull/236

**Comment by msusol** [comment]
Let me look at it in the morning

**Comment by msusol** [comment]
Question for community: Is this just a change for PyPI, or for importing in python as well?

"if the sklearn package is used by one of your dependencies it would be great if you take some time to track which package uses sklearn instead of scikit-learn"

`from sklearn.manifold import MDS, TSNE` -> still valid command?

https://github.com/scikit-learn/sklearn-pypi-package#reason-for-the-deprecation

**Comment by msusol** [comment]
> extract it and locate requirements.txt
> delete sklearn line

The desired deprecation driven change is that `sklearn` is not to be in PyPI (Pipfile), nothing else changes.

I think the bigger issue I found is when I run the tox tests w/sklearn in `requirements.txt` as required, it gives me:

```
from sklearn.manifold import MDS, TSNE
E   ModuleNotFoundError: No module named 'sklearn.manifold'; 'sklearn' is not a package
```
Renaming `sklearn.py` to `lda_ model.py` solves the isssue in tox.

Please let me know what issues you were seeing, specifically.

**Comment by msusol** [comment]
https://github.com/bmabey/pyLDAvis/commit/69a5003bb70a8cbf13898e1a3fda33cc3b2ae63c

(will work on pip package shortly)

**Comment by msusol** [comment]
Resolved in v 3.4.0
